source/index.ts

```diff
@@ -244,6 +260,41 @@ class Conf<T extends Record<string, any> = Record<string, unknown>> implements I
 		return (key as string) in this.store;
 	}
 
+	/**
+	Toggles boolean item.
+
+	@param key - The key of the item to toggle.
+	@returns new value
+	*/
+	toggle<Key extends keyof T>(key: Key | string): boolean {
+		const currentValue = this.has(key) ? this.get(key) : false;
+
+		if (typeof currentValue !== 'boolean') {
+			throw new TypeError(`Expected type to be of type \`boolean\` or empty, is ${typeof currentValue}`);
+		}
+
+		const newValue = !currentValue;
+
+		this.set(key, newValue as T[Key]);
```

Comment:
I would try to remove the force-casting here (the `as`).

Reply:
Doesn't quite work without it. TS doesn't get that it's already tested for types